index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<title>Apple</title>
+	<script src="https://kit.fontawesome.com/1cfd58b03e.js"></script>
+	<link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+	<main class="main-container">
+		<section class="main-section">
+			<div><nav class="navbar">
+			<ul id="global-nav">
+				<li class="navbar-items">
+					<a href="#"><i class="fab fa-apple"></i></a></li>
+				<li class="navbar-items">
+					<a href="#">Store</a></li>
+				<li class="navbar-items">
+					<a href="#">Mac</a></li>
+				<li class="navbar-items">
+					<a href="#">iPod</a></li>
+				<li class="navbar-items">
+					<a href="#">iPhone</a></li>
+				<li class="navbar-items">
+					<a href="#">iPad</a></li>
+				<li class="navbar-items">
+					<a href="#">iTunes</a></li>
+				<li class="navbar-items">
+					<a href="#">Support</a></li>
+				<li class="navbar-items">
+					<input type="text" name="search" class="search-input">
```

Comment:
For the input, you should have the Pseudo function enable in you CSS ex: **.search-input:focus**